generate_stage_repository

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
```

Comment:
Debug left over?
```suggestion
#!/bin/bash -e
```

If it's intentional, I'd instead suggest to enable it after loading settings

```bash
set -x
for os in $OSES; do
  ./build_stage_repository "$PROJECT" "$VERSION" "$os"
done
set +x
```